ops/gauntlet/gauntlet_plus_plus_starknet.go

```diff
@@ -0,0 +1,313 @@
+package gauntlet
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"context"
+	"github.com/rs/zerolog/log"
+	g "github.com/smartcontractkit/gauntlet-plus-plus/sdks/go-gauntlet/client"
+)
+
+var (
+	sgpp *StarknetGauntletPlusPlus
+)
+
+type Request struct {
+	Input map[string]any 		 `json:"input"`
+	Command string					 `json:"command"`
+}
+
+type StarknetGauntletPlusPlus struct {
+	client				*g.ClientWithResponses
```

Comment:
you can run linting with `make format-go` but before that it seems like you need to tidy the go mods with `make gomodtidy` ~ a version conflict came up when I did that so you just need to fix the version bumping